docker/jvm/jsa_launch

```diff
@@ -17,9 +17,9 @@
 KAFKA_CLUSTER_ID="$(opt/kafka/bin/kafka-storage.sh random-uuid)"
 TOPIC="test-topic"
 
-KAFKA_JVM_PERFORMANCE_OPTS="-XX:ArchiveClassesAtExit=storage.jsa" opt/kafka/bin/kafka-storage.sh format --standalone -t $KAFKA_CLUSTER_ID -c opt/kafka/config/kraft/reconfig-server.properties
+KAFKA_JVM_PERFORMANCE_OPTS="-XX:ArchiveClassesAtExit=storage.jsa" opt/kafka/bin/kafka-storage.sh format --standalone -t $KAFKA_CLUSTER_ID -c opt/kafka/config/server.properties
```

Comment:
Not related to this PR but it's a bit odd we don't use absolute paths like `/opt/...` in this script.

Reply:
Yes, it is weird. We should use absolute paths here especially since it's for containers. 🤔 
I will update it in next commit.

Reply:
I see. I will file a jira for this issue. 😺